client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -70,6 +76,34 @@ export default function FiltersPage() {
     return <div>Loading...</div>;
   }
 
+  function findSortType() {
+    if (sortType['value'] == 'alphabetical') {
+      return sortAlphabetically;
+    } else if (sortType['value'] == 'distance') {
+      return sortAlphabetically; // replace with 'sortLocation' once location option is configured
+    }
+  }
+
+  function sortAlphabetically(coop1, coop2) {
+    return coop1['coop_name'] > coop2['coop_name'] ? 1 : -1;
+  }
+
+  function searchCoops(coop) {
+    if (searchInput == null) return true;
+    else {
+      for (var k in coop) {
+        if (
+          typeof coop[k] == 'string' &&
+          k != 'hashed_pass' &&
+          coop[k].includes(searchInput)
```

Comment:
search should be case-insensitive, so convert both to lowercase, i.e. replace with
```js
 coop[k].toLowerCase().includes(searchInput.toLowerCase())